c2corg_api/models/report.py

```diff
@@ -97,7 +97,7 @@ class _ReportMixin(object):
 class Report(_ReportMixin, Document):
```

Comment:
It would be more consistent to rename the file to ``xreports.py``and the class to ``Xreport`` (ditto for archives and mixin classes).